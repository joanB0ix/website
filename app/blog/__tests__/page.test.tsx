import { render } from "@testing-library/react";

import Blog from "../page";

jest.mock("../overview/OverviewSection");

describe("Blog component snapshot", () => {
  it("renders correctly and matches snapshot", () => {
    const { container } = render(<Blog />);
    expect(container).toMatchSnapshot();
  });
});
