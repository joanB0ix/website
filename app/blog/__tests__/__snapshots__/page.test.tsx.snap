// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog component snapshot renders correctly and matches snapshot 1`] = `
<div>
  <main
    class="flex-grow"
  />
</div>
`;
